### ISSUE TITLE
[BUG] Delayed processing with DirectoryWatcher

### ISSUE THREAD

**Issue by efajardo-nv** [nv-morpheus/Morpheus#161]
**Describe the bug**
`CloudTrailSourceStage` and `AppShieldSourceStage` have option (`--watch_directory=True`) to use `DirectoryWatcher` to listen for and process new files as they are added to a directory. There is a bug in the implementation where we see that the first batch of files does not get processed until the second yield from the generator function, [_generate_via_polling](https://github.com/nv-morpheus/Morpheus/blob/33a938c353b23196856ec300790367d057618be4/morpheus/utils/directory_watcher.py#L145) triggered by second batch of files. Therefore, we're always one step behind as new file(s) are added to directory.

**Steps/Code to reproduce bug**
1. Create a watch directory, e.g. `mkdir watch`

2. Start CloudTrail HAMMAH pipeline with just source and train stages with `watch_directory` and `input_glob` options: 
```
morpheus --log_level=DEBUG \
run --num_threads=1 --pipeline_batch_size=1024 --model_max_batch_size=1024 --use_cpp=False \
pipeline-ae --userid_filter=Account-123456789 \
from-cloudtrail --input_glob=watch/hammah-*.csv --max_files=200 --watch_directory=True \
train-ae --train_data_glob=models/datasets/training-data/hammah-*.csv --seed=42
```
3. When you see `====Pipeline Started====`, open a new terminal and go to your watch directory.

4. Copy first input file to watch directory.
```
cp <MORPHEUS_ROOT>/models/datasets/validation-data/hammah-role-g-validation-data.csv .
```
Notice that pipeline does not respond.

5. Copy second input file to watch directory.
```
cp <MORPHEUS_ROOT>/models/datasets/validation-data/hammah-user123-validation-data.csv .
```
Pipeline will respond with following:
```
CloudTrail loading complete. Total rows: 314. Timespan: 2 days 00:42:24
```
The 314 rows corresponds to first input file.

**Expected behavior**
We expect file(s) to be processed as they are added to directory (immediately after corresponding yield from generator function).

**Environment overview (please complete the following information)**
 - Environment location: Docker
 - Method of Morpheus install: Docker image build using `./docker/build_container_dev.sh`

**Environment details**
<details><summary>Click here to see environment details</summary><pre>

     **git***
     commit 33a938c353b23196856ec300790367d057618be4 (HEAD -> branch-22.06, origin/branch-22.06, origin/HEAD)
     Author: David Gardner <96306125+dagardner-nv@users.noreply.github.com>
     Date:   Mon Jun 13 14:09:26 2022 -0700

     Fix for CI check script (#158)

     + Use Github's rest API to determine base branch.
     + Only run check stage on pull requests

     Authors:
     - David Gardner (https://github.com/dagardner-nv)

     Approvers:
     - Michael Demoret (https://github.com/mdemoret-nv)
     - Christopher Harris (https://github.com/cwharris)

     URL: https://github.com/nv-morpheus/Morpheus/pull/158
     **git submodules***

     ***OS Information***
     DISTRIB_ID=Ubuntu
     DISTRIB_RELEASE=20.04
     DISTRIB_CODENAME=focal
     DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
     NAME="Ubuntu"
     VERSION="20.04.4 LTS (Focal Fossa)"
     ID=ubuntu
     ID_LIKE=debian
     PRETTY_NAME="Ubuntu 20.04.4 LTS"
     VERSION_ID="20.04"
     HOME_URL="https://www.ubuntu.com/"
     SUPPORT_URL="https://help.ubuntu.com/"
     BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
     PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
     VERSION_CODENAME=focal
     UBUNTU_CODENAME=focal
     Linux EFAJARDO-DT 5.4.0-113-generic #127~18.04.1-Ubuntu SMP Wed May 18 15:40:23 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

     ***GPU Information***
     Tue Jun 14 00:10:09 2022
     +-----------------------------------------------------------------------------+
     | NVIDIA-SMI 510.47.03    Driver Version: 510.47.03    CUDA Version: 11.6     |
     |-------------------------------+----------------------+----------------------+
     | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
     | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
     |                               |                      |               MIG M. |
     |===============================+======================+======================|
     |   0  Quadro RTX 8000     On   | 00000000:15:00.0 Off |                  Off |
     | 33%   39C    P8    32W / 260W |   2373MiB / 49152MiB |      0%      Default |
     |                               |                      |                  N/A |
     +-------------------------------+----------------------+----------------------+

     +-----------------------------------------------------------------------------+
     | Processes:                                                                  |
     |  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
     |        ID   ID                                                   Usage      |
     |=============================================================================|
     |    0   N/A  N/A       951      C                                    1857MiB |
     |    0   N/A  N/A      1438      G                                       4MiB |
     |    0   N/A  N/A      3920      G                                       4MiB |
     |    0   N/A  N/A     16526      C                                     503MiB |
     +-----------------------------------------------------------------------------+

     ***CPU***
     Architecture:                    x86_64
     CPU op-mode(s):                  32-bit, 64-bit
     Byte Order:                      Little Endian
     Address sizes:                   46 bits physical, 48 bits virtual
     CPU(s):                          12
     On-line CPU(s) list:             0-11
     Thread(s) per core:              2
     Core(s) per socket:              6
     Socket(s):                       1
     NUMA node(s):                    1
     Vendor ID:                       GenuineIntel
     CPU family:                      6
     Model:                           85
     Model name:                      Intel(R) Xeon(R) Gold 6128 CPU @ 3.40GHz
     Stepping:                        4
     CPU MHz:                         1200.101
     CPU max MHz:                     3700.0000
     CPU min MHz:                     1200.0000
     BogoMIPS:                        6800.00
     Virtualization:                  VT-x
     L1d cache:                       192 KiB
     L1i cache:                       192 KiB
     L2 cache:                        6 MiB
     L3 cache:                        19.3 MiB
     NUMA node0 CPU(s):               0-11
     Vulnerability Itlb multihit:     KVM: Vulnerable
     Vulnerability L1tf:              Mitigation; PTE Inversion
     Vulnerability Mds:               Mitigation; Clear CPU buffers; SMT vulnerable
     Vulnerability Meltdown:          Mitigation; PTI
     Vulnerability Spec store bypass: Mitigation; Speculative Store Bypass disabled via prctl and seccomp
     Vulnerability Spectre v1:        Mitigation; usercopy/swapgs barriers and __user pointer sanitization
     Vulnerability Spectre v2:        Mitigation; Retpolines, IBPB conditional, IBRS_FW, STIBP conditional, RSB filling
     Vulnerability Srbds:             Not affected
     Vulnerability Tsx async abort:   Mitigation; Clear CPU buffers; SMT vulnerable
     Flags:                           fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 cdp_l3 invpcid_single pti intel_ppin ssbd mba ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm cqm mpx rdt_a avx512f avx512dq rdseed adx smap clflushopt clwb intel_pt avx512cd avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local dtherm ida arat pln pts hwp hwp_act_window hwp_epp hwp_pkg_req pku ospke md_clear flush_l1d

     ***CMake***
     /opt/conda/envs/morpheus/bin/cmake
     cmake version 3.22.3

     CMake suite maintained and supported by Kitware (kitware.com/cmake).

     ***g++***
     /usr/bin/g++
     g++ (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
     Copyright (C) 2019 Free Software Foundation, Inc.
     This is free software; see the source for copying conditions.  There is NO
     warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

     ***nvcc***
     /usr/local/cuda/bin/nvcc
     nvcc: NVIDIA (R) Cuda compiler driver
     Copyright (c) 2005-2021 NVIDIA Corporation
     Built on Mon_Oct_11_21:27:02_PDT_2021
     Cuda compilation tools, release 11.4, V11.4.152
     Build cuda_11.4.r11.4/compiler.30521435_0

     ***Python***
     /opt/conda/envs/morpheus/bin/python
     Python 3.8.13

     ***Environment Variables***
     PATH                            : /opt/conda/envs/morpheus/bin:/opt/conda/condabin:/opt/conda/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
     LD_LIBRARY_PATH                 : /usr/local/nvidia/lib:/usr/local/nvidia/lib64
     NUMBAPRO_NVVM                   :
     NUMBAPRO_LIBDEVICE              :
     CONDA_PREFIX                    : /opt/conda/envs/morpheus
     PYTHON_PATH                     :

     ***conda packages***
     /opt/conda/condabin/conda
     # packages in environment at /opt/conda/envs/morpheus:
     #
     # Name                    Version                   Build  Channel
     _libgcc_mutex             0.1                 conda_forge    conda-forge
     _openmp_mutex             4.5                       2_gnu    conda-forge
     _sysroot_linux-64_curr_repodata_hack 3                   h5bd9786_13    conda-forge
     abseil-cpp                20211102.0           h27087fc_1    conda-forge
     alabaster                 0.7.12                     py_0    conda-forge
     alembic                   1.8.0              pyhd8ed1ab_0    conda-forge
     appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
     arrow-cpp                 5.0.0           py38h437b12e_25_cuda    conda-forge
     arrow-cpp-proc            3.0.0                      cuda    conda-forge
     asn1crypto                1.5.1              pyhd8ed1ab_0    conda-forge
     atk-1.0                   2.36.0               h3371d22_4    conda-forge
     attrs                     21.4.0             pyhd8ed1ab_0    conda-forge
     autoconf                  2.69            pl5321hd708f79_11    conda-forge
     automake                  1.16.5          pl5321ha770c72_0    conda-forge
     aws-c-cal                 0.5.11               h95a6274_0    conda-forge
     aws-c-common              0.6.2                h7f98852_0    conda-forge
     aws-c-event-stream        0.2.7               h3541f99_13    conda-forge
     aws-c-io                  0.10.5               hfb6a706_0    conda-forge
     aws-checksums             0.1.11               ha31a3da_7    conda-forge
     aws-sdk-cpp               1.8.186              hb4091e7_3    conda-forge
     babel                     2.10.1             pyhd8ed1ab_0    conda-forge
     benchmark                 1.6.1                h9c3ff4c_0    conda-forge
     binutils_impl_linux-64    2.36.1               h193b22a_2    conda-forge
     binutils_linux-64         2.36                hf3e587d_10    conda-forge
     boost-cpp                 1.74.0               h6cacc03_7    conda-forge
     brotli                    1.0.9                    pypi_0    pypi
     brotlipy                  0.7.0           py38h0a891b7_1004    conda-forge
     bzip2                     1.0.8                h7f98852_4    conda-forge
     c-ares                    1.18.1               h7f98852_0    conda-forge
     ca-certificates           2022.5.18.1          ha878542_0    conda-forge
     cached-property           1.5.2                hd8ed1ab_1    conda-forge
     cached_property           1.5.2              pyha770c72_1    conda-forge
     cachetools                5.0.0              pyhd8ed1ab_0    conda-forge
     cairo                     1.16.0            ha12eb4b_1010    conda-forge
     ccache                    4.5.1                haef5404_0    conda-forge
     certifi                   2022.5.18.1      py38h578d9bd_0    conda-forge
     cffi                      1.15.0           py38h3931269_0    conda-forge
     chardet                   4.0.0            py38h578d9bd_3    conda-forge
     charset-normalizer        2.0.12             pyhd8ed1ab_0    conda-forge
     click                     8.1.3            py38h578d9bd_0    conda-forge
     cloudpickle               2.1.0              pyhd8ed1ab_0    conda-forge
     cmake                     3.22.3               h5432695_0    conda-forge
     colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
     configargparse            1.5.3                    pypi_0    pypi
     configparser              5.2.0              pyhd8ed1ab_0    conda-forge
     coverage                  6.4.1            py38h0a891b7_0    conda-forge
     cryptography              37.0.2           py38h2b5fc30_0    conda-forge
     cuda-nvml-dev             11.4.43              he36855d_0    nvidia
     cudatoolkit               11.4.1               h8ab8bb3_9    nvidia
     cudf                      21.10.01        cuda_11.4_py38_hbe913a7_1_morpheus    local
     cudf_kafka                21.10.01        py38_ga1d2d13a14_0    rapidsai
     cupy                      9.3.0            py38ha96c4f3_0    rapidsai
     curl                      7.83.1               h7bff187_0    conda-forge
     cycler                    0.11.0                   pypi_0    pypi
     cyrus-sasl                2.1.27               h230043b_5    conda-forge
     cython                    0.29.24          py38h709712a_1    conda-forge
     databricks-cli            0.12.1             pyhd8ed1ab_0    conda-forge
     datacompy                 0.8.1                    pypi_0    pypi
     dfencoder                 0.0.37                   pypi_0    pypi
     dill                      0.3.5.1                  pypi_0    pypi
     distro                    1.6.0              pyhd8ed1ab_0    conda-forge
     dlpack                    0.5                  h9c3ff4c_0    conda-forge
     docker-compose            1.29.2           py38h578d9bd_1    conda-forge
     docker-py                 5.0.3            py38h578d9bd_2    conda-forge
     docker-pycreds            0.4.0                      py_0    conda-forge
     dockerpty                 0.4.1                      py_0    conda-forge
     docopt                    0.6.2                      py_1    conda-forge
     docutils                  0.18.1                   pypi_0    pypi
     entrypoints               0.4                pyhd8ed1ab_0    conda-forge
     expat                     2.4.8                h27087fc_0    conda-forge
     faker                     12.3.0             pyhd8ed1ab_0    conda-forge
     fastavro                  1.4.12           py38h0a891b7_0    conda-forge
     fastrlock                 0.8              py38hfa26641_2    conda-forge
     flake8                    4.0.1              pyhd8ed1ab_2    conda-forge
     flask                     2.1.2              pyhd8ed1ab_1    conda-forge
     flatbuffers               2.0.6                h27087fc_0    conda-forge
     font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
     font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
     font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
     font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
     fontconfig                2.14.0               h8e229c2_0    conda-forge
     fonts-conda-ecosystem     1                             0    conda-forge
     fonts-conda-forge         1                             0    conda-forge
     fonttools                 4.33.3                   pypi_0    pypi
     freetype                  2.10.4               h0708190_1    conda-forge
     fribidi                   1.0.10               h36c2ea0_0    conda-forge
     fsspec                    2022.5.0           pyhd8ed1ab_0    conda-forge
     gcc_impl_linux-64         9.4.0               h03d3576_16    conda-forge
     gcc_linux-64              9.4.0               h391b98a_10    conda-forge
     gdk-pixbuf                2.42.8               hff1cb4f_0    conda-forge
     gettext                   0.19.8.1          h73d1719_1008    conda-forge
     gevent                    21.12.0                  pypi_0    pypi
     geventhttpclient          1.5.3                    pypi_0    pypi
     gflags                    2.2.2             he1b5a44_1004    conda-forge
     giflib                    5.2.1                h36c2ea0_2    conda-forge
     git                       2.35.3          pl5321h36853c3_0    conda-forge
     gitdb                     4.0.9              pyhd8ed1ab_0    conda-forge
     gitpython                 3.1.27             pyhd8ed1ab_0    conda-forge
     glog                      0.5.0                h48cff8f_0    conda-forge
     gmock                     1.10.0               h4bd325d_7    conda-forge
     graphite2                 1.3.13            h58526e2_1001    conda-forge
     graphviz                  3.0.0                h5abf519_1    conda-forge
     greenlet                  1.1.2            py38hfa26641_2    conda-forge
     grpc-cpp                  1.43.2               h3d78c48_3    conda-forge
     grpcio                    1.46.3                   pypi_0    pypi
     grpcio-channelz           1.46.3                   pypi_0    pypi
     gtest                     1.10.0               h4bd325d_7    conda-forge
     gtk2                      2.24.33              h90689f9_2    conda-forge
     gts                       0.7.6                h64030ff_2    conda-forge
     gunicorn                  20.1.0           py38h578d9bd_2    conda-forge
     gxx_impl_linux-64         9.4.0               h03d3576_16    conda-forge
     gxx_linux-64              9.4.0               h0316aca_10    conda-forge
     harfbuzz                  4.2.0                h40b6f09_0    conda-forge
     icu                       69.1                 h9c3ff4c_0    conda-forge
     idna                      3.3                pyhd8ed1ab_0    conda-forge
     imagesize                 1.3.0              pyhd8ed1ab_0    conda-forge
     importlib-metadata        4.11.4           py38h578d9bd_0    conda-forge
     importlib_resources       5.7.1              pyhd8ed1ab_1    conda-forge
     iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
     isort                     5.10.1             pyhd8ed1ab_0    conda-forge
     itsdangerous              2.1.2              pyhd8ed1ab_0    conda-forge
     jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
     joblib                    1.1.0                    pypi_0    pypi
     jpeg                      9e                   h166bdaf_1    conda-forge
     jsonschema                4.6.0              pyhd8ed1ab_0    conda-forge
     kernel-headers_linux-64   3.10.0              h4a8ded7_13    conda-forge
     keyutils                  1.6.1                h166bdaf_0    conda-forge
     kiwisolver                1.4.2                    pypi_0    pypi
     krb5                      1.19.3               h3790be6_0    conda-forge
     ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
     lerc                      3.0                  h9c3ff4c_0    conda-forge
     libblas                   3.9.0           15_linux64_openblas    conda-forge
     libbrotlicommon           1.0.9                h166bdaf_7    conda-forge
     libbrotlidec              1.0.9                h166bdaf_7    conda-forge
     libbrotlienc              1.0.9                h166bdaf_7    conda-forge
     libcblas                  3.9.0           15_linux64_openblas    conda-forge
     libcudf                   21.10.01        cuda_11.4_he8700b4_0_morpheus    local
     libcudf_kafka             21.10.01          ga1d2d13a14_0    rapidsai
     libcurl                   7.83.1               h7bff187_0    conda-forge
     libdeflate                1.10                 h7f98852_0    conda-forge
     libedit                   3.1.20191231         he28a2e2_2    conda-forge
     libev                     4.33                 h516909a_1    conda-forge
     libevent                  2.1.10               h9b69904_4    conda-forge
     libffi                    3.4.2                h7f98852_5    conda-forge
     libgcc-devel_linux-64     9.4.0               hd854feb_16    conda-forge
     libgcc-ng                 12.1.0              h8d9b700_16    conda-forge
     libgcrypt                 1.10.1               h166bdaf_0    conda-forge
     libgd                     2.3.3                h283352f_2    conda-forge
     libgfortran-ng            12.1.0              h69a702a_16    conda-forge
     libgfortran5              12.1.0              hdcd56e2_16    conda-forge
     libglib                   2.70.2               h174f98d_4    conda-forge
     libgomp                   12.1.0              h8d9b700_16    conda-forge
     libgpg-error              1.45                 hc0c96e0_0    conda-forge
     libgsasl                  1.10.0               h5b4c23d_0    conda-forge
     libhiredis                1.0.2                h2cc385e_0    conda-forge
     libhwloc                  2.5.0                h6746aa3_0    conda-forge
     libiconv                  1.16                 h516909a_0    conda-forge
     liblapack                 3.9.0           15_linux64_openblas    conda-forge
     libllvm10                 10.0.1               he513fc3_3    conda-forge
     libneo                    22.04.00        cuda_11.4_ga156f61f_0    nvidia/label/dev
     libnghttp2                1.47.0               h727a467_0    conda-forge
     libnsl                    2.0.0                h7f98852_0    conda-forge
     libntlm                   1.4               h7f98852_1002    conda-forge
     libopenblas               0.3.20          pthreads_h78a6416_0    conda-forge
     libpng                    1.6.37               h21135ba_2    conda-forge
     libprotobuf               3.19.4               h780b84a_0    conda-forge
     librdkafka                1.6.1                hc49e61c_1    conda-forge
     librmm                    21.10.01        cuda11.4_gc54767f_0    rapidsai
     librsvg                   2.52.5               h0a9e6e8_3    conda-forge
     libsanitizer              9.4.0               h79bfe98_16    conda-forge
     libssh2                   1.10.0               ha56f1ee_2    conda-forge
     libstdcxx-devel_linux-64  9.4.0               hd854feb_16    conda-forge
     libstdcxx-ng              12.1.0              ha89aaad_16    conda-forge
     libthrift                 0.15.0               he6d91bd_1    conda-forge
     libtiff                   4.4.0                h0fcbabc_0    conda-forge
     libtool                   2.4.6             h9c3ff4c_1008    conda-forge
     libutf8proc               2.7.0                h7f98852_0    conda-forge
     libuuid                   2.32.1            h7f98852_1000    conda-forge
     libuv                     1.43.0               h7f98852_0    conda-forge
     libwebp                   1.2.2                h3452ae3_0    conda-forge
     libwebp-base              1.2.2                h7f98852_1    conda-forge
     libxcb                    1.13              h7f98852_1004    conda-forge
     libxml2                   2.9.12               h885dcf4_1    conda-forge
     libzlib                   1.2.12               h166bdaf_0    conda-forge
     llvmlite                  0.36.0           py38h4630a5e_0    conda-forge
     lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
     m4                        1.4.18            h516909a_1001    conda-forge
     mako                      1.2.0              pyhd8ed1ab_1    conda-forge
     markdown-it-py            2.1.0              pyhd8ed1ab_0    conda-forge
     markupsafe                2.1.1            py38h0a891b7_1    conda-forge
     matplotlib                3.5.2                    pypi_0    pypi
     mccabe                    0.6.1                      py_1    conda-forge
     mdit-py-plugins           0.3.0              pyhd8ed1ab_0    conda-forge
     mdurl                     0.1.0              pyhd8ed1ab_0    conda-forge
     mlflow                    1.26.1           py38he918c71_0    conda-forge
     morpheus                  22.6.0a0+47.g33a938c           dev_0    <develop>
     myst-parser               0.17.0             pyhd8ed1ab_0    conda-forge
     ncurses                   6.3                  h27087fc_1    conda-forge
     neo                       22.04.00        cuda_11.4_py38_ga156f61f_0    nvidia/label/dev
     networkx                  2.8.3                    pypi_0    pypi
     ninja                     1.10.2               h4bd325d_1    conda-forge
     nlohmann_json             3.9.1                h9c3ff4c_1    conda-forge
     nodejs                    17.4.0               h8ca31f7_0    conda-forge
     numba                     0.53.1           py38h8b71fd7_1    conda-forge
     numpy                     1.22.3                   pypi_0    pypi
     nvidia-pyindex            1.0.9                    pypi_0    pypi
     nvtx                      0.2.3            py38h497a2fe_1    conda-forge
     openssl                   1.1.1o               h166bdaf_0    conda-forge
     orc                       1.7.3                h1be678f_0    conda-forge
     ordered-set               4.1.0                    pypi_0    pypi
     packaging                 21.3               pyhd8ed1ab_0    conda-forge
     pandas                    1.3.5            py38h43a58ef_0    conda-forge
     pango                     1.50.7               hbd2fdc8_0    conda-forge
     parquet-cpp               1.5.1                         2    conda-forge
     pcre                      8.45                 h9c3ff4c_0    conda-forge
     pcre2                     10.37                h032f7d1_0    conda-forge
     perl                      5.32.1          2_h7f98852_perl5    conda-forge
     pillow                    9.1.1                    pypi_0    pypi
     pip                       22.1.2             pyhd8ed1ab_0    conda-forge
     pixman                    0.40.0               h36c2ea0_0    conda-forge
     pluggy                    1.0.0            py38h578d9bd_3    conda-forge
     prometheus_client         0.14.1             pyhd8ed1ab_0    conda-forge
     prometheus_flask_exporter 0.20.2             pyhd8ed1ab_0    conda-forge
     protobuf                  3.19.4           py38h709712a_0    conda-forge
     pthread-stubs             0.4               h36c2ea0_1001    conda-forge
     py                        1.11.0             pyh6c4a22f_0    conda-forge
     py-cpuinfo                8.0.0              pyhd8ed1ab_0    conda-forge
     pyarrow                   5.0.0           py38ha746e9d_25_cuda    conda-forge
     pybind11-stubgen          0.10.5             pyhd8ed1ab_0    conda-forge
     pycodestyle               2.8.0              pyhd8ed1ab_0    conda-forge
     pycparser                 2.21               pyhd8ed1ab_0    conda-forge
     pydot                     1.4.2            py38h578d9bd_1    conda-forge
     pyflakes                  2.4.0              pyhd8ed1ab_0    conda-forge
     pygments                  2.12.0             pyhd8ed1ab_0    conda-forge
     pyopenssl                 22.0.0             pyhd8ed1ab_0    conda-forge
     pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
     pyrsistent                0.18.1           py38h0a891b7_1    conda-forge
     pysocks                   1.7.1            py38h578d9bd_5    conda-forge
     pytest                    7.1.2            py38h578d9bd_0    conda-forge
     pytest-benchmark          3.4.1              pyhd8ed1ab_0    conda-forge
     pytest-cov                3.0.0              pyhd8ed1ab_0    conda-forge
     python                    3.8.13          h582c2e5_0_cpython    conda-forge
     python-confluent-kafka    1.6.0            py38h497a2fe_1    conda-forge
     python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
     python-dotenv             0.20.0             pyhd8ed1ab_0    conda-forge
     python-graphviz           0.20               pyhaef67bd_0    conda-forge
     python-rapidjson          1.6                      pypi_0    pypi
     python_abi                3.8                      2_cp38    conda-forge
     pytz                      2022.1             pyhd8ed1ab_0    conda-forge
     pyyaml                    6.0              py38h0a891b7_4    conda-forge
     querystring_parser        1.2.4                      py_0    conda-forge
     rapidjson                 1.1.0             he1b5a44_1002    conda-forge
     re2                       2022.02.01           h9c3ff4c_0    conda-forge
     readline                  8.1                  h46c0cb4_0    conda-forge
     requests                  2.27.1             pyhd8ed1ab_0    conda-forge
     rhash                     1.4.1                h7f98852_0    conda-forge
     rmm                       21.10.01        cuda_11.4_py38_gc54767f_0    rapidsai
     s2n                       1.0.10               h9b69904_0    conda-forge
     scikit-build              0.13.1             pyhca92ed8_0    conda-forge
     scikit-learn              0.23.1                   pypi_0    pypi
     scipy                     1.8.1            py38h1ee437e_0    conda-forge
     setuptools                59.8.0           py38h578d9bd_1    conda-forge
     six                       1.16.0             pyh6c4a22f_0    conda-forge
     smmap                     3.0.5              pyh44b312d_0    conda-forge
     snappy                    1.1.9                hbd366e4_1    conda-forge
     snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
     spdlog                    1.8.5                h4bd325d_1    conda-forge
     sphinx                    4.5.0              pyh6c4a22f_0    conda-forge
     sphinx_rtd_theme          1.0.0              pyhd8ed1ab_0    conda-forge
     sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
     sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
     sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
     sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
     sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
     sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_2    conda-forge
     sqlalchemy                1.4.37           py38h0a891b7_0    conda-forge
     sqlite                    3.38.5               h4ff8645_0    conda-forge
     sqlparse                  0.4.2              pyhd8ed1ab_0    conda-forge
     sysroot_linux-64          2.17                h4a8ded7_13    conda-forge
     tabulate                  0.8.9              pyhd8ed1ab_0    conda-forge
     tenacity                  8.0.1              pyhd8ed1ab_0    conda-forge
     tensorboardx              2.5.1                    pypi_0    pypi
     texttable                 1.6.4              pyhd8ed1ab_0    conda-forge
     threadpoolctl             3.1.0                    pypi_0    pypi
     tk                        8.6.12               h27826a3_0    conda-forge
     toml                      0.10.2             pyhd8ed1ab_0    conda-forge
     tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
     torch                     1.10.2+cu113             pypi_0    pypi
     tqdm                      4.64.0             pyhd8ed1ab_0    conda-forge
     tritonclient              2.17.0                   pypi_0    pypi
     typing-extensions         4.2.0                hd8ed1ab_1    conda-forge
     typing-utils              0.1.0                    pypi_0    pypi
     typing_extensions         4.2.0              pyha770c72_1    conda-forge
     ucx                       1.12.0+gd367332      cuda11.2_0    rapidsai
     urllib3                   1.26.9             pyhd8ed1ab_0    conda-forge
     watchdog                  2.1.8                    pypi_0    pypi
     websocket-client          1.3.2              pyhd8ed1ab_0    conda-forge
     werkzeug                  2.1.2              pyhd8ed1ab_1    conda-forge
     wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
     xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
     xorg-libice               1.0.10               h7f98852_0    conda-forge
     xorg-libsm                1.2.3             hd9c2040_1000    conda-forge
     xorg-libx11               1.7.2                h7f98852_0    conda-forge
     xorg-libxau               1.0.9                h7f98852_0    conda-forge
     xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
     xorg-libxext              1.3.4                h7f98852_1    conda-forge
     xorg-libxrender           0.9.10            h7f98852_1003    conda-forge
     xorg-renderproto          0.11.1            h7f98852_1002    conda-forge
     xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
     xorg-xproto               7.0.31            h7f98852_1007    conda-forge
     xtensor                   0.24.2               h924138e_0    conda-forge
     xtl                       0.7.4                h4bd325d_0    conda-forge
     xz                        5.2.5                h516909a_1    conda-forge
     yaml                      0.2.5                h7f98852_2    conda-forge
     zipp                      3.8.0              pyhd8ed1ab_0    conda-forge
     zlib                      1.2.12               h166bdaf_0    conda-forge
     zope-event                4.5.0                    pypi_0    pypi
     zope-interface            5.4.0                    pypi_0    pypi
     zstd                      1.5.2                h8a70e8d_1    conda-forge

</pre></details>

**Comment by dagardner-nv** [comment]
@efajardo-nv  this looks like it could be caused by #330

**Comment by dagardner-nv** [comment]
PR #399 should fix this then.

**Comment by dagardner-nv** [comment]
@efajardo-nv PR #399 has been merged, can you update your conda env and confirm this is fixed now?

**Comment by efajardo-nv** [comment]
@dagardner-nv confirmed that this is fixed by PR #399. closing issue. thanks.
